CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ Nothing yet
 
 - - -
 
+## 0.3.1 (2015-10-14)
+* updating bower package prefs to ignore moved pldoc assets
+
+- - -
+
+## 0.3.0 (2015-10-09)
```

Comment:
**NOTE**: Added in missing CHANGELOG notes from https://github.com/edx/ux-pattern-library/pull/178